Content.Server/Radiation/Systems/RadiationSystem.GridCast.cs

```diff
@@ -218,4 +232,18 @@ private void UpdateGridcast()
 
         return ray;
     }
+
+    private float GetAdjustedRadiationIntensity(EntityUid uid, float rads)
+    {
+        var radblockingComps = new List<RadiationBlockingContainerComponent>();
+        if (_container.TryFindComponentsOnEntityContainerOrParent(uid, _radiationBlockingContainers, radblockingComps))
+        {
+            rads -= radblockingComps.Sum(x => x.RadResistance);
+
+            if (rads <= 0)
+                rads = 0;
+        }
+
+        return rads;
```

Comment:
just clamp rads to 0 at the end, instead of doing it here.

Reply:
Done